src/pages/products/Products.module.css

```diff
@@ -68,17 +68,22 @@
   .grid__cols__auto {
     grid-template-columns: repeat(auto-fit, minmax(18rem, 1fr));
   }
+
+  .desktop__grid {
+    display: grid;
+    grid-template-columns: 1fr 3fr;
+  }
 }
 
 /* Media queries to handle upto large screens */
-@media screen and (max-width: 768px) {
+@media screen and (min-width: 651px) and (max-width: 768px) {
```

Comment:
media queries can also be bundled together in a single file.

Reply:
It made more sense to keep the media queries in the same CSS file where the styles are defined.
We should discuss this if it is good practice